### PR TITLE
User agent tracking for client SDKs

### PR DESCRIPTION
**Pull request by robertjd**

### CODE REVIEW

File: single-page-apps.md [hunk 1]
```diff
@@ -118,6 +118,28 @@ Please see [login][] and [registration][] for specific information on each
...
+* In the front-end library, if a request to the framework is on the same domain
+  (won't cause a cross-domain error), the library should add the following
+  header:
+
+  > Stormpath-Agent: stormpath-sdk-angularjs/0.9.0
+
+* If the `Stormpath-Agent` header is seen on an incoming request, the framework
+  integration should pass this value down to the SDK when making any REST API
+  requests that are needed to fulfill the framework response.  The SDK should
+  append this value to the `User-Agent` header that is send to the Stormpath
```
Comment (rdegges) [inline single-page-apps.md]: *sent

**Comment by omgitstom** [comment]
Change it to be:

`X-Stormpath-Agent`

**Comment by robertjd** [comment]
Changes: Prepend the value.  Break this out from SPA document, as we need to do this for native mobile as well

**Comment by nbarbettini** [comment]
:+1:

**Comment by robertjd** [comment]
Updated with our fixes from today's meeting.

**Pull request merged**